src/Drawer/drawer.module.scss

```diff
@@ -77,6 +76,23 @@ $drawer-width: 315px;
 
 .close {
   @include closeX();
+  padding: 0px;
+
+  &-icon-wrapper {
+    display: flex;
+    padding: 10px;
+    border-radius: 50px;
+    border: 1px solid $gray-6;
+  }
+
+  &:focus {
```

Comment:
Do we want this focus on all "X"s? If so, I would add this to the `closeX` mixin

Reply:
That's a good idea, i've made that change now.